src/components/side-panel/layer-panel/layer-panel.tsx

```diff
@@ -140,6 +140,7 @@ function LayerPanelFactory(
       const {layer, datasets, isDraggable, layerTypeOptions} = this.props;
       const {config} = layer;
       const {isConfigActive} = config;
+      const allowDuplicate = typeof layer.isValidToSave === 'function' && layer.isValidToSave();
```

Comment:
Originally this line was
```js
const allowDuplicate = typeof layer.isValidToSave === 'function' && layer.isValidToSave() && isValid;
```
but `isValid` now doesn't exist on the layer type definition. @heshan0131 do you think this diff will work as intended?

Reply:
make a note in `[feat] Disable a layer after an error in Deck (#180)` to address it when cherrypick

Reply:
✅